etc/greenboot/motd/redboot.motd

```diff
@@ -0,0 +1 @@
+Health Check FAILED! Boot Status is RED!
```

Comment:
Those aren't too useful yet though, given that the eventual goal is to reboot if we have a red boot. So you'd never actually see this, right?

Though I can picture it being useful if we develop this to show e.g. the number of reboots that were done before reaching a greenboot, or whether the default boot failed and we had to rollback, etc...

I guess what I think I'm saying is, we should eventually be generating `/run/greenboot/motd` dynamically? Instead of linking static files into there.

Reply:
I guess so, sounds sensible